app/models/data-operation.ts

```diff
@@ -175,6 +203,25 @@ export interface SRS {
   epsg?: string;
 }
 
+/**
+ * Transform a related URL from CMR to a Harmony related URL that can be used in the DataOperation.
+ * @param relatedUrl - A CmrRelatedUrl
+ * @returns the equivalent HarmonyRelatedUrl 
+ */
+export function cmrRelatedUrlToHarmony(relatedUrl: CmrRelatedUrl): HarmonyRelatedUrl {
+  const harmonyRelatedUrl: HarmonyRelatedUrl = {
+    url: relatedUrl.URL,
+    urlContentType: relatedUrl.URLContentType,
+    type: relatedUrl.Type,
+  };
+  if (relatedUrl.Subtype) harmonyRelatedUrl.subtype = relatedUrl.Subtype;
```

Comment:
Wondering why we can't assign these potentially undefined/null fields and need to perform these checks.

Edit: I'm guessing the JSON passed in the message was including things like `{ 'Description': null }`.

Reply:
Yep -- I was thinking it'd be best to not end up with properties like { 'Description': undefined } in the final data operation.

Reply:
I think it's still redundant, though:
```
> JSON.stringify({foo: undefined, bar: null})
'{"bar":null}'
```

Reply:
I'm happy to get rid of it if that's more ideal.

Reply:
Got rid of this code.